packages/webview_flutter/webview_flutter_web/lib/webview_flutter_web.dart

```diff
@@ -0,0 +1,182 @@
+        }
+        final IFrameElement element =
+            document.getElementById('webview-$viewId')! as IFrameElement;
+        if (creationParams.initialUrl != null) {
+          element.src = creationParams.initialUrl;
+        }
+        onWebViewPlatformCreated(WebWebViewPlatformController(
+          element,
+        ));
+      },
+    );
+  }
+
+  @override
+  Future<bool> clearCookies() async => false;
+
+  /// Gets called when the plugin is registered.
+  static void registerWith(Registrar registrar) {}
+}
+
+/// Implementation of [WebViewPlatformController] for web.
+class WebWebViewPlatformController implements WebViewPlatformController {
+  /// Constructs a [WebWebViewPlatformController].
+  WebWebViewPlatformController(this._element);
+
+  final IFrameElement _element;
+
+  @override
+  Future<void> addJavascriptChannels(Set<String> javascriptChannelNames) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<bool> canGoBack() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<bool> canGoForward() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> clearCache() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<String?> currentUrl() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<String> evaluateJavascript(String javascript) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<int> getScrollX() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<int> getScrollY() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<String?> getTitle() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> goBack() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> goForward() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> loadUrl(String url, Map<String, String>? headers) async {
+    _element.src = url;
+  }
+
+  @override
+  Future<void> reload() {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> removeJavascriptChannels(Set<String> javascriptChannelNames) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> runJavascript(String javascript) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<String> runJavascriptReturningResult(String javascript) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> scrollBy(int x, int y) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> scrollTo(int x, int y) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> updateSettings(WebSettings setting) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> loadFile(String absoluteFilePath) {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> loadHtmlString(
+    String html, {
+    String? baseUrl,
+  }) async {
+    throw UnimplementedError();
+  }
+
+  @override
+  Future<void> loadRequest(WebViewRequest request) {
+    throw UnimplementedError();
```

Comment:
Shouldn't this at least trivial `get` requests, and only throw for the rest, so that this package doesn't stop working once the app-facing package changes over to loadRequest?

Reply:
Yes. When I wrote this implementation this was not yet part of the platform interface, so I had not yet gotten around to it. It's now implemented together with `loadHtmlString` as well.